doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -158,6 +158,10 @@ nRF5340 Audio
 nRF Desktop
 -----------
 
+* Added:
+
+   * Application-specific Kconfig options to simplify the Debug configurations for the Logging and Shell subsystems.
```

Comment:
Maybe we could link to the 2 newly introduced options (or to the new documentation section) here?

Reply:
sure